memcached.c

```diff
@@ -8261,13 +8588,26 @@ int main (int argc, char **argv) {
         }
         ext_storage = storage;
         /* page mover algorithm for extstore needs memory prefilled */
-        slabs_prefill_global();
+        prefill = true;
     }
 #endif
 
     if (settings.drop_privileges) {
         setup_privilege_violations_handler();
     }
+
+    if (prefill)
+        slabs_prefill_global();
```

Comment:
Should be a no-op in that case. Think I want to leave it if memory adjustment at startup ever happens, since it might end up with extra space to allocate. Otherwise prefill happens for cold-start-restart and extstore, so it's correct otherwise. I think.